# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../mapnik/libtool-fix.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- mapnik-0.5.1.orig/include/mapnik/plugin.hpp 2008-09-17 20:46:56.598535454 +0200
+++ mapnik-0.5.1/include/mapnik/plugin.hpp      2008-09-17 20:47:29.278542547 +0200
@@ -29,7 +29,8 @@
 // stl
 #include <string>

-typedef struct lt_dlhandle_struct * lt_dlhandle;
+#include <ltdl.h>
+
 namespace mapnik
 {
    class PluginInfo : boost::noncopyable
